setup.cfg

```diff
@@ -5,3 +5,16 @@ versionfile_source = napari/_version.py
 versionfile_build = napari/_version.py
 tag_prefix = v
 parentdir_prefix = napari-
+
+[flake8]
+# Ignores - https://lintlyci.github.io/Flake8Rules
+# E203  Whitespace before ':'  (sometimes conflicts with black)
+# E501 line too long (84 > 79 characters)  (sometimes too annoying)
+# W503	Line break occurred before a binary operator
+# E711	Comparison to none should be 'if cond is none:'
+# E712	Comparison to true should be 'if cond is true:' or 'if cond:'
```

Comment:
Good call waiting on the first round of review to fix these, but I agree with @bjlittle (:wave:!) that we should definitely tackle them!